servers/physics_3d/godot_shape_3d.h

```diff
@@ -256,7 +256,7 @@ class GodotCylinderShape3D : public GodotShape3D {
 	_FORCE_INLINE_ real_t get_height() const { return height; }
 	_FORCE_INLINE_ real_t get_radius() const { return radius; }
 
-	virtual real_t get_area() const override { return 4.0 / 3.0 * Math_PI * radius * radius * radius + height * Math_PI * radius * radius; }
+	virtual real_t get_volume() const override { return 4.0 / 3.0 * Math_PI * radius * radius * radius + height * Math_PI * radius * radius; }
```

Comment:
Not relevant to the goals of this PR, but this calculation looks wrong.
```suggestion
	virtual real_t get_volume() const override { height * Math_PI * radius * radius; }
```

Reply:
Moved to a separate issue: #54636